pkg/rancher-desktop/components/DashboardOpen.vue

```diff
@@ -0,0 +1,35 @@
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+
+import { State as K8sState } from '@pkg/backend/backend';
+
+export default Vue.extend({
+  name:     'dashboard-button',
+  computed: {
+    ...mapGetters('preferences', ['getPreferences']),
+    ...mapGetters('k8sManager', { k8sState: 'getK8sState' }),
+    kubernetesEnabled(): boolean {
+      return this.getPreferences.kubernetes.enabled;
+    },
+    kubernetesStarted(): boolean {
+      return this.k8sState === K8sState.STARTED;
+    },
+  },
+  methods: {
+    openDashboard() {
+      this.$emit('open-dashboard');
+    },
+  },
+});
+</script>
+
+<template>
+  <button
+    v-if="kubernetesEnabled && kubernetesStarted"
+    class="btn role-secondary btn-icon-text"
```

Comment:
We don't have an icon here; is `btn-icon-text` still the correct class? (The same probably applies to the other button)

Reply:
The `btn-icon-text` style doesn't do anything too much to have a icon. May be renaming the style to just `btn-text` is something I can think of but I would like to leave it as it is just in case we want to bring the icon back at some point for whatever reason.